src/Installer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Composer\Script\Event;
+use RecursiveIteratorIterator;
+
+final class Installer
+{
+    public static function postUpdate(Event $event = null): void
+    {
+        self::chmodRecursive('runtime', 0777);
+    }
+
+    private static function chmodRecursive(string $path, int $mode): void
+    {
+        chmod($path, $mode);
+        $iterator = new RecursiveIteratorIterator(new \RecursiveDirectoryIterator($path), RecursiveIteratorIterator::SELF_FIRST);
```

Comment:
```suggestion
        $iterator = new RecursiveIteratorIterator(
            new \RecursiveDirectoryIterator(
                $path,
                \FilesystemIterator::SKIP_DOTS | \FilesystemIterator::CURRENT_AS_PATHNAME
            ),
            RecursiveIteratorIterator::SELF_FIRST
        );
        foreach ($iterator as $item) {
            chmod($item, $mode);
        }
```

Reply:
you can also add short aliases for longnamed classes like `use FilesystemIterator as FSIterator; use RecursiveDirectoryIterator as DirIterator`...